apps/miniapp/domains/common/utils/oidcAuth.tsx

```diff
@@ -40,6 +46,8 @@ const OidcAuthProvider = ({ children }) => {
     const errorReason = get(error, 'errorReason')
     const errorData = isObject(error) ? JSON.stringify(error) : '<NonObject>'
     const { user, isLoading: isUserLoading, isAuthenticated } = useAuth()
+    const [authInProgress, setAuthInProgress] = useState(false)
+    const [, setAuthError] = useState(false)
```

Comment:
Is this state necessary?
Every call of `setAuthError` causes re-rendering, but the value from this state is never used.

Reply:
Please see the note above. It's just an example. We should redesign miniapp, since right now it's just a legacy mess. I'll do it outside this PR